src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Cmdlets/Item/GetAzureRmRecoveryServicesBackupItem.cs

```diff
@@ -52,6 +53,14 @@ public class GetAzureRmRecoveryServicesBackupItem : RSBackupVaultCmdletBase
         [ValidateNotNullOrEmpty]
         public BackupManagementType BackupManagementType { get; set; }
 
+        /// <summary>
+        /// When this option is specified, only those items which belong to this policy will be returned.
+        /// </summary>
+        [Parameter(Mandatory = true, Position = 1, HelpMessage = ParamHelpMsgs.Policy.ProtectionPolicy,
```

Comment:
Use ParamHelpMsgs.Item.ProtectionPolicy and use the text provided by the PM in both the help and in summary.